numpyro/primitives.py

```diff
@@ -304,10 +304,11 @@ def _subsample(name, size, subsample_size, dim):
         }
         apply_stack(msg)
         subsample = msg['value']
+        subsample_size = msg['args'][1]  # TODO: rewrite plate
```

Comment:
I guess this is not relevant to this PR but this change could be useful and seems to make plate message more flexible. +1

Btw, what does this TODO mean?

Reply:
I originally added it to use the `change_subsample_size` handler to avoid raising an error (current fix is to change to [warning](https://github.com/pyro-ppl/numpyro/blob/2c261739513eae2205e7ae17ccfa8ffeca991371/numpyro/primitives.py#L309)).

The TODO has been resolved and not relevant to current implementation. I'll remove it.